.circleci/regenerate.py

```diff
@@ -247,7 +246,7 @@ def unittest_workflows(indentation=6):
                 if device_type == "gpu":
                     if python_version != "3.8":
```

Comment:
Because of this line unittest_linux_gpu_py3.8 failes. Not sure what the logic behind this.

Reply:
Do you have an issue to track? If so, please mention it in the comment